cloudconvert.json

```diff
@@ -0,0 +1,689 @@
+{
+    "appid": "5cf7f9aa-877a-45fb-8384-362d2b935db9",
+    "name": "Cloud Convert",
+    "description": "This app supports executing investigative and generic type of actions to convert the SOAR vault files to various formats",
+    "type": "endpoint",
+    "product_vendor": "CloudConvert",
+    "logo": "logo_cloudconvert.svg",
+    "logo_dark": "logo_cloudconvert_dark.svg",
+    "product_name": "CloudConvert",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
+    "license": "Copyright (c) 2022 Splunk Inc.",
+    "app_version": "1.0.0",
+    "utctime_updated": "2022-05-05T11:38:35.889474Z",
+    "package_name": "phantom_cloudconvert",
+    "main_module": "cloudconvert_connector.py",
+    "min_phantom_version": "5.2.0",
+    "fips_compliant": true,
+    "app_wizard_version": "1.0.0",
+    "latest_tested_version": [
+        "Cloud, 20 June, 2022"
+    ],
+    "configuration": {
+        "api_key": {
+            "description": "Cloud Convert API Key",
+            "data_type": "password",
+            "required": true,
+            "order": 0
+        },
+        "timeout": {
+            "description": "Number of minutes to poll for converted file (Default: 1)",
+            "data_type": "numeric",
+            "default": 1,
+            "order": 1
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate the asset configuration for connectivity using supplied configuration",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "convert file",
+            "identifier": "convert_file",
+            "description": "Convert one filetype to another",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "vault_id": {
+                    "description": "Vault ID",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "vault id"
+                    ],
+                    "order": 0
+                },
+                "filetype": {
+                    "description": "Type of the file in output",
+                    "data_type": "string",
+                    "value_list": [
```

Comment:
How is this list ordered? Might be useful to do alphabetical or most commonly used at top

Reply:
There are no such commonly used outputs formats. Hence, we used an API call https://api.cloudconvert.com/v2/convert/formats provided by cloud convert which returns the list of valid output formats and placed them in the value_list in the same order. It isn't mentioned anywhere whether this API returns most commonly used formats first or not. Also as value list provides valid suggestions for providing some input(For instance, On entering 'p' in dropdown parameter, all values in list starting with 'p' are returned), we haven't changed the list returned by the API.